cvat/rq_patching.py

```diff
@@ -69,7 +71,22 @@ def custom_started_job_registry_cleanup(self, timestamp: Optional[float] = None)
 
     return job_ids
 
+def custom_hearbeat(self, timestamp: datetime, ttl: int, pipeline: Optional['Pipeline'] = None, xx: bool = False):
+    self.last_heartbeat = timestamp
+    connection = pipeline if pipeline is not None else self.connection
+    connection.hset(self.key, 'last_heartbeat', utcformat(self.last_heartbeat))
+
+    # the only difference with the default implementation
+    # is to avoid adding job to started job registry
+    # that is not clear why they adds a job to started_job_registry when the job has failed
+    # also, as far as I see they also commented this line in unrealeased (2.0) version
+    exc_info = sys.exc_info()
+    is_stopped_export_job = isinstance(exc_info[1], (StopRequested, SystemExit))
```

Comment:
It is not clear why this condition means that an `export` job has been stopped (and generally it should not mean that). Consider using a more abstract name.